src/lang/en.json

```diff
@@ -514,6 +516,7 @@
     "passwordNotMatchMsg": "The repeat password does not match.",
     "notificationDescription": "Notifications must be assigned to a monitor to function.",
     "keywordDescription": "Search keyword in plain HTML or JSON response. The search is case-sensitive.",
+    "jsonQueryDescription": "Do a json Query against the response and check for expected value.",
```

Comment:
Please add a better description what a json-query is and link to the packages docs you used for further details.
Imo this is currently not clear enough

Reply:
Done.